src/content/blog/astro-5.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: "Astro 5.0"
+description: "Astro 5.0 brings exciting new features like the Astro Content Layer for seamless content organization from any source, and Server Islands for effortlessly combining static and dynamic personalized content."
```

Comment:
```suggestion
description: "Astro 5.0 brings exciting new features like the Astro Content Layer for seamless content loading from any source, and Server Islands for effortlessly combining static and dynamic personalized content."
```
I think we were talking about hitting "loading" as the key idea that's new here?